File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 2/ex21.c
#include <stdio.h>

main()
{
    printf("Precedencia de um produto\n\n");

    //vars
    float preco;
    int codigo;

    //Entrada
    printf("Insira o preco do produto: R$ ");
    scanf("%f",&preco);

    printf("Insira o codigo de precedencia do produto: ");
    scanf("%d",&codigo);

    //Processamento
    if ( codigo == 1 )
        printf("Precedencia: Sul");
    else if( codigo == 2  )
        printf("Precedencia: Norte");
    else if ( codigo == 3 )
        printf("Precedencia: Leste");
    else if ( codigo == 4 )
        printf("Precedencia: Oeste");
    else if ( codigo == 5 || codigo == 6 )
        printf("Precedencia: Nordeste");
    else if ( codigo == 7 || codigo == 8 || codigo == 9 )
        printf("Precedencia: Sudeste");
    else if ( codigo >= 10 || codigo <= 20 )
        printf("Precedencia: Centro-oeste");
    else
        printf("Precedencia: Nordeste");
}

File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 2/src/Ex21.java
//Importando Scanner
import java.util.Scanner;

public class Ex21 {

	public static void main(String[] args){
		
		//Instanciando Scanner
		Scanner ent = new Scanner(System.in);
		
		System.out.println("Procedência de um produto\n\n");
		
		//vars
		float preco;
		int codigo;
		
		//Entrada
		System.out.print("Insira o preço do produto: R$ ");
		preco = ent.nextFloat();
		
		System.out.print("Insira o código de procedência do produto: ");
		codigo = ent.nextInt();
		
		//Processamento
		if ( codigo == 1 )
			System.out.println("Procedência: Sul");
		else if ( codigo == 2 )
			System.out.println("Procedência: Norte");
		else if ( codigo == 3 )
			System.out.println("Procedência: Leste");
		else if ( codigo == 4 )
			System.out.println("Procedência: Oeste");
		else if ( codigo==5 || codigo==6 )
			System.out.println("Procedência: Nordeste");
		else if ( codigo>=7 && codigo<=9 )
			System.out.println("Procedência: Sudeste");
		else if ( codigo>=10 && codigo<=20 )
			System.out.println("Procedência: Centro-oeste");
		else
			System.out.println("Procedência: Nordeste");
		
		
		
	}
}
